### PR TITLE
[bde driver] black list linux_kernel_bde driver

### PR DESCRIPTION
**Pull request by yxieca**
Signed-off-by: Ying Xie <ying.xie@microsoft.com>

**- What I did**
This driver should be loaded by sonic service. If kernel tries to load
it, the driver would be loaded with default parameters, which is not
right for sonic.

### CODE REVIEW

**Comment by yxieca** [comment]
retest this please

File: files/build_templates/sonic_debian_extension.j2 [hunk 1]
```diff
@@ -261,6 +262,9 @@ fi
 ## copy platform rc.local
 sudo cp $IMAGE_CONFIGS/platform/rc.local $FILESYSTEM_ROOT/etc/
 
+## copy blacklist file
+sudo cp $IMAGE_CONFIGS/platform/linux_kernel_bde.conf $FILESYSTEM_ROOT/etc/modprobe.d/
```
Comment (lguohan) [inline files/build_templates/sonic_debian_extension.j2]: is this broadcom platform only?

File: files/image_config/platform/linux_kernel_bde.conf [hunk 2]
```diff
@@ -0,0 +1 @@
+blacklist linux_kernel_bde
```
Comment (zhenggen-xu) [inline files/image_config/platform/linux_kernel_bde.conf]: who is trying to load linux_kernel_bde if we are not running sonic service? 
backlist does not prevent it from being loaded as a dependency for another kernel module that is not blacklisted, are there any other modules depending on this module that could still start this?

Comment (lguohan) [reply files/image_config/platform/linux_kernel_bde.conf]: it is automatically loaded by kernel based on the pcie id. @jipanyang , user-bde and bnet bde are not pci driver, they will not be automatically loaded.

**Pull request merged**